### ISSUE TITLE
[CTA Section]: React wrapper VO does not read the horizontal rule

### ISSUE THREAD

**Issue by oliviaflory** [carbon-design-system/carbon-for-ibm-dotcom#8129]
### Description

Voice over is not reading the CTA block border in the CTA section component. 

In other components that have a horizontal rule, the rule is read out as "Horizontal splitter" this is helpful for the user to know when a section is ending a new one is beginning.
Example: [Content block](https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/react/?path=/story/components-content-block--default)

### Component(s) impacted

CTA Section
- simple
- with content items
- with link list

Must turn border knob on

### Browser

Chrome

### Carbon for IBM.com version

v1.15.0 rc 0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-react

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components-react/?path=/story/components-cta-section--simple

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
